### PR TITLE
Refactor logging functionality

### PR DESCRIPTION
**Pull request by rliu-parasoft**

### CODE REVIEW

**Comment by rwang-parasoft** [comment]
Need to verify the changes aftect the logging function in Jenkins plugin.

**Comment by rliu-parasoft** [comment]
> Need to verify the changes aftect the logging function in Jenkins plugin.

**Comment by rliu-parasoft** [comment]
> Need to verify the changes aftect the logging function in Jenkins plugin.

After the change, it works as usual in the Jenkins plugin

**Review by rwang-parasoft — APPROVED** [review]

File: src/main/java/com/parasoft/findings/utils/common/logging/ILoggerHandler.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
+/*
```
Comment (yshi-parasoft) [inline src/main/java/com/parasoft/findings/utils/common/logging/ILoggerHandler.java:1]: Why do we need `ILoggerHandler `and `ILoggerHandlerFactory `as interfaces when they are only implemented by one class? Are we planning to implement them with more classes?

**Pull request merged**